xtask/src/bootstrap_groth16.rs

```diff
@@ -158,19 +178,22 @@ fn bootstrap_test_receipt() {
     let journal = format!("bytes public constant JOURNAL = hex\"{journal}\";");
     let image_id = format!("bytes32 public constant IMAGE_ID = bytes32({image_id});");
 
+    let solidity_test_receipt_path = risc0_ethereum_path.join(SOLIDITY_TEST_RECEIPT_PATH);
     let content =
         &format!("{SOL_HEADER}{LIB_HEADER}\n{seal}\n{post_digest}\n{journal}\n{image_id}\n}}");
-    fs::write(SOLIDITY_TEST_RECEIPT_PATH, content).expect(&format!(
-        "failed to save changes to {}",
-        SOLIDITY_TEST_RECEIPT_PATH
-    ));
+    fs::write(&solidity_test_receipt_path, content).unwrap_or_else(|_| {
+        panic!(
+            "failed to save changes to {}",
+            solidity_test_receipt_path.display()
+        )
+    });
 
     // Use forge fmt to format the file.
     Command::new("forge")
```

Comment:
By the way, I had issues with this on my `lxc` machine because on Linux, `sudo` is required to run docker commands, and I didn't install `forge` in the `PATH` variable for `root`. It's a bit of a catch 22, and I ended up getting around it by commenting out all the `format` calls. Not really sure how to resolve that, and it doesn't need to be resolved in this PR as long as it works in CI.

Reply:
I see, the easiest would be to add your user permission to run docker:
```console
sudo groupadd docker
sudo usermod -aG docker $USER
```
https://docs.docker.com/engine/install/linux-postinstall/

Reply:
Oh nice, that's a good tip... kinda scary too since talking to Docker gives trivial access to `root` 😅